### ISSUE TITLE
A new bug!

### ISSUE THREAD

**Issue by Javajava1** [milvus-io/bootcamp#658]
Hello,I found a trouble that might be a new bug.I planed to stand up milvus server with code.So I came in bootcamp-master\solutions\reverse_image_search\quick_deploy\server\src,  edited ./config.py and ran ./main.py.The output information said all were well.Then I put into a image to get similar ones.However,no matter how much top k similarity I choosed,all returns were ten pictures(the first picture),the output information was OK(the second picture) .Moreover,about 10 mins later, there were some output information:500 internal error!(the third picture).In fact,the reason I choose to rum milvus server with code was that I wanted to use pre-existing mysql:5.7. So I exported Mysql_user,Mysql_pwd and ran milvus server with docker command.However ,it didnot work.The logs about milvus server was connection refuesed(the fourth picture).But when I ran with code,it worked well(the first picture).What problem was that? What should I do? Your help is important to me.I will be appreciated. 

In short,I just only got 10 pictures no matter how much top k similarity I choosed.I ran milvus server the code.

**To Reproduce**
- Which solution are you running? reverse_image_search;https://github.com/milvus-io/bootcamp/tree/master/solutions/reverse_image_search/quick_deploy

- Steps to reproduce the behavior(Docker or Source code):
1. Go to 'bootcamp-master\solutions\reverse_image_search\quick_deploy\server\src'
2. Edit config.py
3. python main.py
![image](https://user-images.githubusercontent.com/29594737/133594235-80161d04-2eac-4083-8616-233fc639676a.png)

**Expected behavior**
The number of pictures is different when I choose different top k similarity.

**Screenshots**
<img width="1335" alt="3a6f08879fa1f5c7df62a6febc97b48" src="https://user-images.githubusercontent.com/29594737/133594758-66659d76-05b7-47dd-9609-e7702453e6e4.png">
![image](https://user-images.githubusercontent.com/29594737/133594920-0c0366d4-b3c7-4c8f-b591-071712565149.png)
<img width="794" alt="fc9be94e3f855622fb3ec06b4a4a14a" src="https://user-images.githubusercontent.com/29594737/133594933-5468d6d9-2db2-4d24-909c-9003ac9de04e.png">
![image](https://user-images.githubusercontent.com/29594737/133595006-944a280a-fd8c-452c-8d1e-1768fbfd9af3.png)

**Software version (please complete the following information):**
 - Milvus: [e.g. 2.0.0rc6]
 - Server: [e.g. 2.0.0]
 - Client: [e.g. 1.0]

**Comment by shiyu22** [comment]
How many images did you insert into the service? How about searching for another image, is it still 10 results?

**Comment by Javajava1** [comment]
Yes,the result is always 10 pictures from 17125 picture collection, no matter what picture I put in,how much top k I choose .

**Comment by Javajava1** [comment]
I asked this question in wechat group.One administrator said it was found first time in running server with code.

**Comment by shiyu22** [comment]
Thank you, get it. I think it's really a TOPK bug for reverse image search. 

It looks like reverse-image-search 2.0 misses the TOPK parameter, so the server calls the default `TOP_K=10`. You can fix this by modifying and running the source code. You need to add the TOPK parameter to main.py when [searching](https://github.com/milvus-io/bootcamp/blob/master/solutions/reverse_image_search/quick_deploy/server/src/main.py#L101), like `from config import TOP_K` and `topk: int=TOP_K`. BTW, the TOPK parameter of the [reverse image search 1.0](https://github.com/milvus-io/bootcamp/tree/1.0/solutions/pic_search) is valid. 

If you modify the code and it works, you are more than welcome to [submit a PR](https://github.com/milvus-io/bootcamp/pulls) to Bootcamp as a contributor :)

**Comment by Javajava1** [comment]
Dear Shiyu,thank for you reply,I will try to fix this problem.

**Comment by RafaelDSS** [comment]
I fixed the bug. I will send a pull request
